src/main/java/io/appform/dropwizard/actors/actor/HaMode.java

```diff
@@ -0,0 +1,28 @@
+package io.appform.dropwizard.actors.actor;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.Map;
+
+public enum HaMode {
+    EXACTLY {
+        @Override
+        public Map<String, Object> handleConfig(ActorConfig actorConfig) {
```

Comment:
Visitor pattern are used for this purpose to segregate code from the enum class

Reply:
Have separated the implementation out in a different class.